Develop/assets/css/style.css

```diff
@@ -101,6 +121,18 @@ code {
   color: var(--heliotrope);
   border: 2px solid var(--gin);
   transition: all 0.5s ease-in-out;
+
+  /* new */
+  position: relative;
+  margin-top: -4rem;
+  width: 95%;
+  border-radius: 0.5rem;
+  padding: 0.5rem;
+}
```

Comment:
Positions the "Heading box" of each card relative to the section and by default layers it above the outline. 
Negative margin pulls it up ontop the outline.